src/constants/map.js

```diff
@@ -345,9 +345,9 @@ export const register = (tool, annotationConstructor) => {
     currentPalette: availablePalettes[0],
     availablePalettes,
     toolNames: [toolName],
-    annotationCheck: annotationConstructor
-      ? annotation => annotation instanceof annotationConstructor
-      : null,
+    annotationCheck: customAnnotCheckFunc ?
+      annotation => customAnnotCheckFunc(annotation) :
```

Comment:
I think in this case we can just return `customAnnotCheckFunc`, no need to create another function that just pass the argument to `customAnnotCheckFunc`. But not a big deal.